lib/Perspective/Config.php

```diff
@@ -253,16 +253,18 @@ public static function getRuntimePerspective(\Pimcore\Model\User $currentUser =
     protected static function getRuntimeElementTreeConfig($name)
     {
         $masterConfig = self::get()->toArray();
-
-        $config = $masterConfig[$name];
+        
+        $config = $masterConfig[$name] ?? null;
         if (!$config) {
             $config = [];
         }
-
-        $tmpResult = $config['elementTree'];
-        if (is_null($tmpResult)) {
-            $tmpResult = [];
+        
+        if (!array_key_exists('elementTree', $config)) {
+            return [];
         }
```

Comment:
This is not necessary anymore (and probably wrong anyways)
```suggestion
        
        if (!array_key_exists('elementTree', $config)) {
            return [];
        }
```

Reply:
GitHub-Web messed it up ^^ should be fixed now :)